Training2/src/task1_2/src/App.js

```diff
@@ -0,0 +1,33 @@
+import "./App.css";
+import CloseButton from "./CloseButton";
+import { useState } from "react";
+import Modal from "./Modal";
+
+function App() {
+  const [isClosed, setIsClosed] = useState(false);
+  const handleClick = () => {
+    setIsClosed(true);
+    console.log("hi", isClosed);
+  };
+  return (
+    <div className="App">
+      <h1>TRANING2 TASK 1+2</h1>
+      {isClosed === false ? (
+        <Modal>
```

Comment:
@phuongluongZigvy  the Modal should have the props i`sClosed` an `toggleIsClosed` itself

Reply:
I fixed it by triggering function handleClick from Modal to App so that the App know when it will render the "Open Model" button and close Modal

Reply:
I fixed it by triggering function handleClick from Modal to App so that the App know when it will render the "Open Model" button and close Modal